### ISSUE TITLE
Includes of SC, Understanding, and definitions from WCAG are not working

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#84]
With the change to markdown, we need some Bikeshed scripting magic to incorporate the original text from the include files that WCAG has in their GitHub repository. Specifically to include:
- SC text ([GitHub location for SC text](https://github.com/w3c/wcag/tree/main/guidelines/sc)
- Definitions text ([GitHub location for the definitions](https://github.com/w3c/wcag/tree/main/guidelines/terms))
- Understanding text, just the Intent section ([GitHub location for Understanding documents](https://github.com/w3c/wcag/tree/main/understanding))

**Comment by shawnthompson** [comment]
I'm looking into it and reached out to @sidvishnoi in the respec slack channel. 

Basically how to include inside an include.

**Comment by shawnthompson** [comment]
Once https://github.com/w3c/respec/pull/4372 is merged we can start including from includes. 

I can continue this work by first adding all the `<blockquote>` back from the [index.html](https://github.com/w3c/wcag2ict/blob/b46a97d1e727f61628a64ae40658ef124cb4a7cd/index.html) we had before the change to markdown files.

**Comment by maryjom** [comment]
I met with @michael-n-cooper today and he's working on this task on 14 April. Hoping that work will close out this issue.

**Comment by maryjom** [comment]
Includes for SC and definition text are now working. The includes of the Understanding Intent was moved to the document formatting Issue.

**Comment by maryjom** [comment]
This issue is being handled with issues #118 and #147.
